targets/bech32.cpp

```diff
@@ -20,10 +20,8 @@ std::string Bech32Core(Span<const uint8_t> buffer)
 }
 
 
-void Bech32(FuzzedDataProvider& provider) 
+FUZZ_TARGET(Bech32)
```

Comment:
```suggestion
FUZZ_TARGET(bech32)
```

same for the others?